### DISCUSSION TITLE
After following the steps of integration, I am getting "Build input file cannot be found: '/Users/{path-to-project-derived-data}/Build/Products/Debug-iphoneos/json.o'. Did you forget to declare this file as an output of a script phase or custom build rule which produces it?"

### DISCUSSION THREAD

**Discussion by fattysun1121** [nlohmann/json#4405] [Q&A]
<img width="921" alt="Screenshot 2024-06-26 at 9 47 54 AM" src="https://github.com/nlohmann/json/assets/85023400/31d69f2a-b1da-4130-bbfd-9f1fbd0f4179">
<img width="981" alt="Screenshot 2024-06-26 at 9 49 13 AM" src="https://github.com/nlohmann/json/assets/85023400/e0b2d0ff-68ec-4d5e-b78d-46841c56173b">
<img width="924" alt="Screenshot 2024-06-26 at 9 49 29 AM" src="https://github.com/nlohmann/json/assets/85023400/063990ed-46db-436a-aae3-bfba4383251d">

I followed the above steps to integrate nlohmann json to our project, did I miss anything?

**Comment by nlohmann** [comment]
The information you provided are insufficient to diagnose the issue. The screen shot looks like Xcode - what are you trying to build there?

**Comment by HamidAliSE** [comment]
@fattysun1121 May be [this solution](https://stackoverflow.com/a/79930740/32627764) could help.
